fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -0,0 +1,28 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.catalog;
+
+import com.starrocks.common.io.Writable;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+
+public abstract class Catalog implements Writable {
```

Comment:
do we need extend MetaObject?

Reply:
Not yet

Reply:
@gengjun-git does MetaObject necessary for object need to persist?